docs/manual/06_Environments.md

```diff
@@ -0,0 +1,278 @@
+# Environments: Running your application
+
+Clusterous is designed around Docker, which is a technology and suite of tools for running lightweight application "containers".
+
+This document is a guide to running your Docker-based cluster application on a Clusterous cluster.
+
+## Prerequisites
+
+This guide assumes you are familiar with Docker and have already made your application run in a Docker container in a test environment. Refer to Appendix [TODO: LINK] for information on getting started with Docker.
+
+## Background
+In order to deploy an parallel compute application on to a compute cluster, a number of actions must take place. They including many or all of the following:
+
+- building an updated Docker image (or images) for your application and making them available on the cluster
+- copying any additional files that your application needs, but hasn't been included in the Docker images. These may include configuration files, launch scripts, input data, etc.
+- running all the Docker containers correctly, across all the nodes in the cluster, with each using the appropriate resources, etc.
+- a connection between the client machine and the cluster; if your application has a web based interface (e.g. the Jupyter notebook) or any other kind of external interface, you need to be able to easily access it
+
+The Clusterous Environment file feature provides an easy way to achieve all of the above with a single step.
+
+
+
+Once you have made your application run inside a Docker container, you need to provide Clusterous a way of deploying it on to the cluster. This is done by means of an environment file that describes how to do so.
+
+An environment file is a special YAML file that provides Clusterous instructions to build the Docker images for your application, copy files to the cluster, run containers correctly, and create an SSH tunnel if necessary.
+
+## Running the Docker application in Clusterous
+There are two things to keep in mind when making your application run correctly in a Clusterous cluster.
+
+- 
+- All Clusterous clusters have a dedicated "shared volume", which is an NFS volume accessible to all nodes. This volume is mounted in /home/data, and is typically used for launch scripts, common configuration files and so on, as well as for your application's input/output data.
+
```

Comment:
That section had some broken paragraphs. I have reworded and added a couple of new sections.